internal/pkg/git/git.go

```diff
@@ -72,10 +72,11 @@ func Checkout(ctx context.Context, url, ref string, sparse bool, logger log.Logg
 		if strings.Contains(result.stdErr, fmt.Sprintf("Remote branch %s not found", r.ref)) {
 			return nil, fmt.Errorf(`reference "%s" not found in the git repository "%s"`, r.ref, r.url)
 		}
-		if result.exitCode == 128 {
-			return nil, fmt.Errorf(`git repository could not be checked out from "%s": %s`, r.url, result.stdErr)
```

Comment:
Medzi touto hlaskou, a dalsou `cannot checkout git repository` nevidim nejaky zasadny rozdiel ... no niekedy to na Windows spadne s nespravnym kodom.